docs/content/reference/smoke-test/_index.md

```diff
@@ -0,0 +1,65 @@
+---
+type: "docs"
+title: "Smoke Test"
+linkTitle: "Smoke Test"
+weight: 40
+description: >
+    Smoke test for Drasi Installation
+---
+
+
+### Prerequisites
+- [Kubectl](https://kubernetes.io/docs/tasks/tools/)
+- [Drasi CLI](/reference/command-line-interface/)
+- A Kubernetes cluster with an instance of Drasi deployed
+
+### Deploying the Smoke Test
+To deploy the smoke test, execute the following command.
+
+#### If you deployed Drasi to the default `drasi-system` namespace:
+{{< tabpane langEqualsHeader=true >}}
+{{< tab header="MacOS/VS Code Dev Container" lang="Bash" >}}
+curl -fsSL https://raw.githubusercontent.com/drasi-project/drasi-platform/main/dev-tools/smoke-tests/smoke-test.sh | /bin/bash
+{{< /tab >}}
+{{< tab header="Windows PowerShell" lang="Bash" >}}
+Invoke-Command -ScriptBlock ([scriptblock]::Create([System.Text.Encoding]::UTF8.GetString((New-Object Net.WebClient).DownloadData('https://raw.githubusercontent.com/drasi-project/drasi-platform/main/dev-tools/smoke-tests/smoke-test.ps1')))) 
+{{< /tab >}}
+{{< /tabpane >}}
+
+#### If Drasi was deployed to a different namespace:
+Replace `<namespace>` with the namespace that you installed Drasi in.
+{{< tabpane langEqualsHeader=true >}}
+{{< tab header="MacOS/VS Code Dev Container" lang="Bash" >}}
+curl -fsSL https://raw.githubusercontent.com/drasi-project/drasi-platform/main/dev-tools/smoke-tests/smoke-test.sh | /bin/bash -s <namespace>
+{{< /tab >}}
+{{< tab header="Windows PowerShell" lang="Bash" >}}
+Invoke-Command -ScriptBlock ([scriptblock]::Create([System.Text.Encoding]::UTF8.GetString((New-Object Net.WebClient).DownloadData('https://raw.githubusercontent.com/drasi-project/drasi-platform/main/dev-tools/smoke-tests/smoke-test.ps1')))) -ArgumentList <namespace>
+{{< /tab >}}
+{{< /tabpane >}}
+
+
```

Comment:
Need a section in here that describes what they will see when the run the test and it is successful, and what they might see of the test fails.